src/ServerConfigParams.ts

```diff
@@ -183,15 +177,6 @@ export async function resolveServerConfig(
   );
   /*  } */
 
-  if (
-    !(await contractInteractor.isContractDeployed(
-      contractsConfig.relayHubAddress
-    ))
-  ) {
-    error(
-      `RelayHub: no contract at address ${contractsConfig.relayHubAddress}`
-    );
-  }
   if (appConfig.url == null) error('missing param: url');
   if (appConfig.workdir == null) error('missing param: workdir');
 
```

Comment:
I know we didn't introduce it in these changes, but I think the line `return { ...serverDefaultConfiguration, ...contractsConfig, ...appConfig };` may be wrong, because we're flattening the configuration object.

Reply:
its weird, I remember that I had this already fixed in the PR that I closed and I just copy and paste it, since before returning the config I was validating it.

Reply:
updated